Signal/src/views/ReminderView.swift

```diff
@@ -126,6 +124,9 @@ class ReminderView: UIView {
     }
 
     @objc func handleTap(gestureRecognizer: UIGestureRecognizer) {
-        tapAction()
+        guard gestureRecognizer.state == .recognized else {
```

Comment:
I've always wanted to ask you, since you often insert this incantation - when would this method get called when the gesture state is not recognized?

Reply:
`UIGestureRecognizers` have a state.  Their `action` is called whenever the state changes and your handler is supposed to respond to that state.  The exception is `UITapGestureRecognizer`.  For some reason (presumably simplicity over consistency), Apple was inconsistent with this GR.  Tap GRs only invoke their action when they recognize.   So this incantation isn't strictly necessary, but I always add it in case they ever decide to make tap GRs consistent with the others, since tap handlers should only do their work when the GR is in the "recognized" state.